NOTICE.BINARY

```diff
@@ -2277,41 +1574,6 @@ see CREDITS file.
 
 
 
-================= Jetty 6.1.26 =================
-jetty-sslengine-6.1.26.jar
-jetty-util-6.1.26.jar
-=================
-==============================================================
- Jetty Web Container
- Copyright 1995-2009 Mort Bay Consulting Pty Ltd
-==============================================================
-
-The Jetty Web Container is Copyright Mort Bay Consulting Pty Ltd
-unless otherwise noted. It is dual licensed under the apache 2.0
-license and eclipse 1.0 license. Jetty may be distributed under
-either license.
-
-The javax.servlet package used was sourced from the Apache
-Software Foundation and is distributed under the apache 2.0
-license.
-
-The UnixCrypt.java code implements the one way cryptography used by
```

Comment:
I think we have to keep this UnixCrypt part for jetty 6.1.26, these jars are bundled along with the hadoop-client 2.8.3 dependency

Reply:
correction, the jar that has UnixCrypt class is from druid-kerberos

druid-kerberos/jetty-6.1.26.jar

Reply:
Thanks, added. Also removed "Xerces2 Java Parser 2.9.1".